### PR TITLE
Styles and icons

### PR DESCRIPTION
**Pull request by indirectlylit**
* include material icons as font
* add button styling, based on [purecss](https://github.com/yahoo/pure/blob/master/src/buttons/css/buttons-core.css)
* autoprefix external styles
* improved styling of links

### CODE REVIEW

**Comment by indirectlylit** [comment]
this should be ready for review & merge

File: kolibri/core/assets/src/core-global.styl [hunk 1]
```diff
@@ -19,19 +19,83 @@
...
+  -ms-interpolation-mode: bicubic
+
+
+// https://davidwalsh.name/html5-boilerplate
+svg:not(:root)
+  overflow: hidden
+
+
+button
+  // structure (from https://github.com/yahoo/pure/blob/master/src/buttons/css/buttons-core.css)
```
Comment (rtibbles) [inline kolibri/core/assets/src/core-global.styl:63]: Fine with this for now - @66eli77 @whitzhu do you want to weigh in on whether you would prefer materialize based buttons instead? http://materializecss.com/buttons.html

File: kolibri/core/assets/src/core_app_instance.js [hunk 2]
```diff
@@ -3,6 +3,9 @@
 require('normalize.css');
 require('./font-NotoSans.css');
 require('./core-global.styl');
+require('webpack-material-design-icons');
```
Comment (rtibbles) [inline kolibri/core/assets/src/core_app_instance.js:6]: The use of this here is what makes me wonder if we might prefer material design buttons atop?

**Comment by rtibbles** [comment]
Looks decent, will wait a couple of hours for comment on the places I mentioned then, merge.

File: kolibri/core/assets/src/core-global.styl (hunk 1 above)
Comment (indirectlylit) [inline kolibri/core/assets/src/core-global.styl:63]: I tried to implement the button in our [style guide](https://github.com/learningequality/Kolibri_style_guide) as minimally as possible, but I'm open to alternatives.

I looked at using parts of [mdl light](https://getmdl.io/) but it seemed overkill to style a button.

**Comment by rtibbles** [comment]
Another concern - it makes the Explore page go from this:
image

to this:

image

Centre panel is now left aligned, and a scrollbar has appeared on the container for the breadcrumb.

File: kolibri/core/assets/src/core-global.styl (hunk 1 above)
Comment (indirectlylit) [inline kolibri/core/assets/src/core-global.styl:63]: (I had also had trouble getting the button to look good - still not too happy with it)

**Comment by indirectlylit** [comment]
can you check if that happens in the roll-up PR? aspects may have been addressed by the margins and grid work.

The scrollbar on the breadcrumbs is a known (to me) but I was waiting to address in separate work

**Comment by rtibbles** [comment]
Sure, will review code by PR and behaviour by rollup then.

**Comment by rtibbles** [comment]
One other note, I'm getting a lint warning:
```
kolibri/core/assets/src/core-global.styl
  line 24:  * selector is disallowed  *, *:before, *:after

  ⚠  1 warning
```

Could you add an explanatory comment and lint disabling for this line so as not to startle future devs? :)

**Pull request merged**